### PR TITLE
Add New Model for enrollments and Changes according to the dev server issues.

### PR DESCRIPTION
**Pull request by imhassantariq**

### CODE REVIEW

File: models.py [hunk 1]
```diff
@@ -29,3 +29,14 @@ class EdxNodeBBCategory(models.Model):
 
     def __str__(self):
         return str(self.course_key)
+
+
+class EdxNodeBBEnrollment(models.Model):
```
Comment (danialmalik) [inline models.py]: Why have you named this `EdxNodeBBEnrollment` when  you are only saving only edx related data. Besides, I think this data can also be retrieved from  edx current db schema.

Comment (imhassantariq) [reply models.py]: This data is saved already in edx database but we have created a new table just to be sure that the user is successfully joined in a group of that course on nodebb. We save into this table when we get OK response from nodebb join group call.

We can make changes here by changing the field from edx_user to `nodebb_uid` and `course_id` to nodebb `group_slug` but that doesn't make any changes I think. in both cases, we are saving data to make sure our edx and nodebb are completely synced.

Comment (danialmalik) [reply models.py]: I get your point but this class name is confusing.

File: client/groups.py [hunk 2]
```diff
@@ -31,7 +35,7 @@ def create(self, course_id, **payload):
             tuple: Tuple in the form (response_code, json_response) received from requests call.
         """
         response_code, json_response = self.post('/api/v2/groups', **payload)
-
+
```
[the remark this answers is not in the document]
Comment (imhassantariq) [reply client/groups.py]: Done

**Pull request merged**